src/drivers/boards/px4fmu-v4/board_config.h

```diff
@@ -86,12 +86,17 @@
 
 #define GPIO_SPI_CS_FRAM		(GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTD|GPIO_PIN10)
 
+#define GPIO_SPI_CS_BMI055_ACC  (GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTC|GPIO_PIN15)
```

Comment:
@scheiermanns I think that suggestion makes sense. We're configuring the same GPIO twice, which can be hard to maintain since its unexpected.